docs/GSMTC-Support-And-Popular-Apps.md

```diff
@@ -22,26 +21,30 @@ So, let's check the compatibility or support report.
 - 🟪 - Feature support is incomplete i.e. in-progress or being improved or expected to arrive soon or temporarily unavailable
 - 🟥 - Feature not supported
 
-| Application | Support | Play\Pause | Previous | Next | Thumbnail | Media Title | Media Artist | App Info | Shuffle | Repeat | Stop | Timeline information |
-| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
-| **Browsers** |
-| Chromium and its derivatives (such as **Chrome**, **New MS Edge**, **Opera**, **Brave**, **Vivaldi** and etc.) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
-| Firefox | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
-| **Media players** |
-| *(i) Music Only* |
-| Spotify | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟩 | 🟩 | 🟩 | 🟩 |
-| Groove Music | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 |
-| [Dopamine-Windows](https://github.com/digimezzo/dopamine-windows) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟩 | 🟩 | 🟩 | 🟪 ([In-Progress](https://github.com/digimezzo/dopamine-windows/pull/1206)) |
+| Application | Support | Play/Pause | Previous | Next | Thumbnail | Media Title | Media Artist | App Info | Shuffle | Repeat | Stop | Timeline information |
+| --- | --- | ---| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+||| **BROWSERS** |
+| Chromium based | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
+| Firefox        | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
+||| **MEDIA** | **PLAYERS**|
+||
+| Spotify        | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟩 | 🟩 | 🟩 | 🟩 |
+| Groove Music   | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟥 | 🟥 |
```

Comment:
Bruh, why spaces only on some places? It's inconsistent. Could you remove this please?

Reply:
The spaces dont appear in the doc.
Adding spaces in the raw looks good.

Reply:
There are some big gaps after this, so did not do so for others.

Reply:
Hmmm. I know.